analyses/independent-samples/00-repeated-samples.Rmd

```diff
@@ -26,7 +31,11 @@ set.seed(201910)
 
 ```{r load files}
 root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+if(params$base_run == 0) {
```

Comment:
Same comment here about if statements and making it so you don't have to repeat the same code.